samza-core/src/main/java/org/apache/samza/runtime/ApplicationRunnerMain.java

```diff
@@ -59,7 +61,7 @@ public static void main(String[] args) throws Exception {
 
     switch (op) {
       case RUN:
-        appRunner.run();
+        appRunner.run(null);
```

Comment:
I think can revert all changes in this class.

Reply:
Nvm, we didn't keep the old method in RemoteApplicationRunner. This is fine.

Reply:
I need to remove the unused imports here. I'm just running a local check before pushing the change.